src/test/java/com/conveyal/gtfs/GTFSFeedTest.java

```diff
@@ -152,4 +169,114 @@ public void canDoRoundtripLoadAndWriteToZipFile() throws IOException {
             );
         }
     }
+
+    /**
+     * Make sure the correct timezone of a stop is returned
+     */
+    @Test
+    public void canGetAgencyTimeZoneForStop() {
+        GTFSFeed feed = GTFSFeed.fromFile(simpleGtfsZipFileName);
+        assertThat(
+            feed.getAgencyTimeZoneForStop("4u6g").getId(),
+            equalTo("America/Los_Angeles")
+        );
+    }
+
+    /**
+     * Make sure that a GTFS feed with interpolated stop times have calculated times after feed processing
+     * @throws GTFSFeed.FirstAndLastStopsDoNotHaveTimes
+     */
+    @Test
+    public void canGetInterpolatedTimes() throws GTFSFeed.FirstAndLastStopsDoNotHaveTimes, IOException {
+        String tripId = "a30277f8-e50a-4a85-9141-b1e0da9d429d";
+
+        String gtfsZipFileName = TestUtils.zipFolderFiles("fake-agency-interpolated-stop-times");
+
+        GTFSFeed feed = GTFSFeed.fromFile(gtfsZipFileName);
+        Iterable<StopTime> stopTimes = feed.getInterpolatedStopTimesForTrip(tripId);
```

Comment:
I'm not even sure if this method is really used anywhere anymore.  It might not be worth developing a test for.  @abyrd, is this used anywhere in r5?

Reply:
If something isn't used anymore we should remove it from the codebase.